### DISCUSSION TITLE
Feature: Scale up the chart on large high-res screens

### DISCUSSION THREAD

**Discussion by Mshmaitelly** [blackadderkate/weather-widget-2#175] [Ideas]
4K High Res monitors are widely available. Is it possible to have the option to scale up the drop-down chart for the weather forecast on such high-res monitors? The fonts are barely visible. 
Thanks a lot for such nice widget.

**Comment by blackadderkate** [comment]
Unfortunately I don't have a high-res monitor - which obviously makes development and testing this kind of request pretty difficult. 😉

Once I've nailed down the remaining issues for Plasma 6, got it released on the widget store and fixed the inevitable bugs that get reported from that, I'll see what I can do - I might be able to setup a VM with a really high resolution or something.
